client/components/page/index.css

```diff
@@ -8,3 +10,8 @@
 .PageContent {
   clear: both;
 }
+
+.viewer {
```

Comment:
[nit] Just an observation. I've scrolled about 1/8th the way through this pull request, and this is the first non-whitespace change that I've seen. As a general rule, avoid making unnecessary whitespaces changes in your PRs so the reviewers can save time and focus on the important edits that contain functional changes.

If things are so messy that making whitespaces changes is important, it's best to contain them to a separate commit that can be easily scanned over by reviewers.

Reply:
Definitely agree with this; the whitespace changes were part of the merge conflicts.